controllers/gce/loadbalancers/loadbalancers.go

```diff
@@ -743,6 +742,12 @@ func (l *L7) UpdateUrlMap(ingressRules utils.GCEURLMap) error {
 		}
 		l.um.PathMatchers = append(l.um.PathMatchers, pathMatcher)
 	}
+	oldMap, _ := l.cloud.GetUrlMap(l.um.Name)
```

Comment:
can you please write a unittest like (or modify the same): https://github.com/kubernetes/ingress/blob/master/controllers/gce/loadbalancers/loadbalancers_test.go#L149 to make sure it no-ops when there's no change?